utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -228,50 +260,24 @@ int main(int argc, char **argv)
     // Initialize roi that can be updated in case-wise augmentations if needed
     RpptROI roi;
 
-    // Set maxHeight, maxWidth and ROIs for src/dst
-    const int images = noOfImages;
-
-    for(int i = 0; i < imageNames.size(); i++)
-    {
-        string temp = inputPath;
-        temp += imageNames[i];
-        if (layoutType == 0 || layoutType == 1)
-            image = imread(temp, 1);
-        else
-            image = imread(temp, 0);
-
-        roiTensorPtrSrc[i].xywhROI = {0, 0, image.cols, image.rows};
-        roiTensorPtrDst[i].xywhROI = {0, 0, image.cols, image.rows};
-        dstImgSizes[i].width = roiTensorPtrDst[i].xywhROI.roiWidth;
-        dstImgSizes[i].height = roiTensorPtrDst[i].xywhROI.roiHeight;
-
-        maxHeight = std::max(maxHeight, roiTensorPtrSrc[i].xywhROI.roiHeight);
-        maxWidth = std::max(maxWidth, roiTensorPtrSrc[i].xywhROI.roiWidth);
-        maxDstHeight = std::max(maxDstHeight, roiTensorPtrDst[i].xywhROI.roiHeight);
-        maxDstWidth = std::max(maxDstWidth, roiTensorPtrDst[i].xywhROI.roiWidth);
-
-        count++;
-    }
-
-    // Check if any of maxWidth and maxHeight is less than or equal to 0
-    if(maxHeight <= 0 || maxWidth <= 0)
-    {
-        std::cerr<<"Unable to read images properly.Please check the input path of the files specified";
-        exit(0);
-    }
-
     Rpp32u outputChannels = inputChannels;
     if(pln1OutTypeCase)
         outputChannels = 1;
     Rpp32u offsetInBytes = 0;
 
     // Set numDims, offset, n/c/h/w values, strides for src/dst
-    set_descriptor_dims_and_strides(srcDescPtr, noOfImages, maxHeight, maxWidth, inputChannels, offsetInBytes);
-    set_descriptor_dims_and_strides(dstDescPtr, noOfImages, maxDstHeight, maxDstWidth, outputChannels, offsetInBytes);
+    set_descriptor_dims_and_strides(srcDescPtr, batchSize, MAX_HEIGHT, MAX_WIDTH, inputChannels, offsetInBytes);
```

Comment:
So is the row stride always 3840 to jump to the new row even if all images in a batch are no more than 224?
Can allocation of buffers be based on the max size supported 3840x2160, but stride setting be done for each batch?

Reply:
we can discuss this in a call